ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/EthProtocol.java

```diff
@@ -125,4 +125,8 @@ public static EthProtocol get() {
   public static boolean isEth66Compatible(final Capability capability) {
     return NAME.equals(capability.getName()) && capability.getVersion() >= ETH66.getVersion();
   }
+
+  public static Capability getLatestVersion() {
+    return ETH68;
```

Comment:
nit: we can also create a `LATEST` static final variable towards the top of the class and then return that variable here. For example:
```
public static final Capability ETH67 = Capability.create(NAME, EthProtocolVersion.V67);
  public static final Capability ETH68 = Capability.create(NAME, EthProtocolVersion.V68);
  public static final Capability LATEST = ETH68;
```